client/src/mapView.js

```diff
@@ -10,15 +10,57 @@ var MapView = React.createClass({
     router: React.PropTypes.func
   },
 
-  render () {
+  // this is called after the first reder of the component
+  componentDidMount () {
```

Comment:
for future reference: what if I go back to the homeview and come back with new start and end coordinates? How will we update the map?